python/lsst/ts/standardscripts/base_track_target_and_take_image.py

```diff
@@ -180,6 +183,7 @@ async def configure(self, config):
         self.config.az_wrap_strategy = getattr(
             self.tcs.WrapStrategy, self.config.az_wrap_strategy
         )
+        self.note = getattr(self.config, "note", None)
```

Comment:
Really minor comment for you to consider.

We keep the configuration in `self.config` already so, in principle, you don't need to add a `self.note`. However, if you want to make it easier to reach this parameter, consider making this a property;

```
@property
def note(self):
    return getattr(self.config, "note", None) if self.config is not None else None
```

Reply:
Adopted.